src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  background: #fff;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  width: 100%;
+
+  &__moyo-logo {
+    height: 40px;
+    width: 40px;
+    display: flex;
+    align-items: center;
+  }
+}
+
+.nav__list {
```

Comment:
The styles for '.nav__list' should be inside a separate file named 'nav.scss'. You should create the file, move the styles there, and import it into your main SCSS file.